commit e16aef8b7978a6d5d2ea4c2029bb2bef1b658859
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:22:13 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-nowarn-unsigned-source.diff

diff --git a/src/macports1.0/macports.tcl b/src/macports1.0/macports.tcl
index 40e2c08ec..9ab7e807e 100644
--- a/src/macports1.0/macports.tcl
+++ b/src/macports1.0/macports.tcl
@@ -1264,13 +1264,13 @@ proc mportinit {{up_ui_options {}} {up_options {}} {up_variations {}}} {
                         set sources_default [concat [list $url] $flags]
                     }
                 }
-                if {[string match rsync://*rsync.macports.org/release/ports/ $url]} {
-                    ui_warn "MacPorts is configured to use an unsigned source for the ports tree.\
-Please edit sources.conf and change '$url' to '[string range $url 0 end-14]macports/release/tarballs/ports.tar'."
-                } elseif {[string match rsync://rsync.macports.org/release/* $url]} {
-                    ui_warn "MacPorts is configured to use an older rsync URL for the ports tree.\
-Please edit sources.conf and change '$url' to '[string range $url 0 26]macports/release/tarballs/ports.tar'."
-                }
+#                 if {[string match rsync://*rsync.macports.org/release/ports/ $url]} {
+#                     ui_warn "MacPorts is configured to use an unsigned source for the ports tree.\
+# Please edit sources.conf and change '$url' to '[string range $url 0 end-14]macports/release/tarballs/ports.tar'."
+#                 } elseif {[string match rsync://rsync.macports.org/release/* $url]} {
+#                     ui_warn "MacPorts is configured to use an older rsync URL for the ports tree.\
+# Please edit sources.conf and change '$url' to '[string range $url 0 26]macports/release/tarballs/ports.tar'."
+#                 }
                 switch -- [macports::getprotocol $url] {
                     rsync -
                     https -
